sydent/http/httpsclient.py

```diff
@@ -47,7 +52,7 @@ def __init__(self, sydent):
             #                                                      trustRoot=self.sydent.sslComponents.trustRoot)
             self.agent = Agent(self.sydent.reactor, SydentPolicyForHTTPS(self.sydent))
 
-    def postJson(self, uri, jsonObject):
+    def postJson(self, uri: str, jsonObject: Dict[Any, Any]) -> Optional[Generator]:
```

Comment:
Mypy accepts `Deferred` as long as `Deferred` has no arguments-is it cool to just use `Deferred`? Otherwise it raises `"Deferred" expects no type arguments, but 1 given` error

Reply:
Sure, we'll figure out why the Deferreds aren't subscriptable lataer